src/Linux/local_cache.cpp

```diff
@@ -24,6 +24,9 @@ constexpr locale_t NULL_LOCALE = reinterpret_cast<locale_t>(0);
 static std::string g_cache_dirname = "";
 static std::mutex cache_directory_lock;
 
+static long int last_cache_read_expiry = -1;
+static long int last_cache_read_time = -1;

```

Comment:
I think having these values here for debugging was fine but for production I would recommend exploring all options on how to enable logging on this level. This feels hackish and wil not be clear to future engineers. 

If we end up keeping these we would also have to guarantee thread safety in accessing/writing to them.